src/utils/input_bam.rs

```diff
@@ -21,12 +21,19 @@ pub struct FiberFilters {
         help_heading = "BAM-Options"
     )]
     pub bit_flag: u16,
+    /// Filtering expression to use for filtering records
+    #[clap(
+        global = true,
+        short = 'x',
+        long = "ftx",
+        alias = "ft-expression",
+        default_value = "",
```

Comment:
We should drop this line. Dropping this makes the default None which has nicer ergonomics imo.

Reply:
[50f1012](https://github.com/fiberseq/fibertools-rs/commit/50f101278debedaef50ef82dc39db1d72c565734)